java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -183,4 +183,24 @@ public interface StringCommands {
      * }</pre>
      */
     CompletableFuture<Long> decrBy(String key, long amount);
+
+    /**
+     * Returns the length of the string value stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/strlen/">redis.io</a> for details.
+     * @param key The key to check its length.
+     * @return The length of the string value stored at key.<br>
+     *     If <code>key</code> does not exist, it is treated as an empty string, and the command
+     *     returns <code>0</code>.
+     * @example
+     *     <pre>{@code
+     * client.set("key", "GLIDE").get();
```

Comment:
You don't need this step explicitly we didn't show setups in all other commands.

Reply:
I think it's good to know what is stored at key `"key"`.